### ISSUE TITLE
Sieve filters duplicate user maildirs with legacy short format

### ISSUE THREAD

**Issue by DavidePrincipi** [NethServer/dev#5150]
**Steps to reproduce**

* install roundcube on rc2
* log in as ``first.user``, the ``first.user@domain`` maildir is created under /var/lib/nethserver/vmail/
* from roundcube edit the sieve filters, another maildir ``first.user`` appears under  /var/lib/nethserver/vmail/

**Components**
nethserver-mail-server-1.10.4

Discussion on 

http://community.nethserver.org/t/roundcube-issues-connected-to-ldap/4947

**Comment by DavidePrincipi** [comment]
We can fix with PR https://github.com/NethServer/nethserver-mail-server/pull/15, or by configuring the managesieve service with a poslogin script.

**Comment by DavidePrincipi** [comment]
In nethserver-testing:
nethserver-mail-server-1.10.4-1.2.g5bff66e.ns7.noarch.rpm

**Comment by gsanchietti** [comment]
Logged in with both username forms using roundcubemail:
- giacomo
- giacomo@neth.eu

Sieve filters are always accessible:
```
[root@test ~]# find /var/lib/nethserver/vmail/ -name \*sieve
/var/lib/nethserver/vmail/giacomo@neth.eu/sieve
/var/lib/nethserver/vmail/giacomo@neth.eu/sieve/roundcube.sieve
/var/lib/nethserver/vmail/giacomo@neth.eu/.dovecot.sieve
```

**Comment by gsanchietti** [comment]
Released in 7.3.1611/base:
nethserver-mail-server-1.10.5-1.ns7.noarch.rpm
